src/components/opt-in-flyout/README.md

```diff
@@ -0,0 +1,59 @@
+# Opt-in/Opt-out Flyouts
+
+The `<d2l-labs-opt-in-flyout>` and `<d2l-labs-opt-out-flyout>` can be used in applications to enable users to opt-in or out of new experiences.
+
+<!-- docs: start hidden content -->
```

Comment:
Is this repo hooked up to the Daylight site?  Are you looking to do that as part of this work, or just setting things up nicely in case we one day decide to?